eclair-core/src/test/scala/fr/acinq/eclair/io/PendingChannelsRateLimiterSpec.scala

```diff
@@ -45,6 +49,18 @@ class PendingChannelsRateLimiterSpec extends ScalaTestWithActorTestKit(ConfigFac
   val newChannelIdPrivate1: ByteVector32 = ByteVector32(hex"077777770000000000000000000000000000000000000000000000000000000")
   val channelIdAtLimit1: ByteVector32 = ByteVector32(hex"0888888880000000000000000000000000000000000000000000000000000000")
   val channelIdAtLimit2: ByteVector32 = ByteVector32(hex"0999999990000000000000000000000000000000000000000000000000000000")
+  val waitingForSigs: InteractiveTxSigningSession.WaitingForSigs = {
```

Comment:
Honestly it seems a bit overkill to go through the pain of creating a dummy `DATA_WAIT_FOR_DUAL_FUNDING_SIGNED`, I think we're fine just using the other states that are easier to "mock" in tests.

Reply:
Good point; it was a pita to add and it seems unlikely to catch anything extra - at the cost of readability for the test.
Fixed in [3f71eda](https://github.com/ACINQ/eclair/pull/2687/commits/3f71eda159ac875aa6737564329db1e9f49528c0).